src/System.Net.NameResolution/tests/PalTests/project.json

```diff
@@ -24,12 +24,16 @@
     "System.Text.RegularExpressions": "4.2.0-beta-24403-05",
     "System.Threading": "4.0.12-beta-24403-05",
     "System.Threading.Tasks": "4.0.12-beta-24403-05",
+    "System.Net.Sockets": {
```

Comment:
The change that I am trying to include in this PR is to use an updated version of xunit which depends on NETStandard.Library. Prior to this, we were using a version of xunit that was repackaged to restore for netstandard and did not have NETStandard.Library as a dependency. Now that xunit requires this library, it pulls down various System.Net packages at compile time. 

I think the solution here might be to actually split the tests into two projects and address the two scenarios separately.